### PR TITLE
update deps and fix click for external links in remote substreams docs

### PR DESCRIPTION
**Pull request by dimaMachina**

### CODE REVIEW

File: website/pages/en/substreams/[[...slug]].mdx [hunk 1]
```diff
@@ -41,22 +41,21 @@ export async function getStaticProps({ params: { slug = ['README'] } }) {
     // close unclosed img tags
     .replaceAll(/<img((?:(?!\/>)[^>])*?)>/g, (...m) => `<img${m[1]}/>`)
     // fix gitbook image srcs
-    .replaceAll(/src="[^>"]*\.gitbook\/(.*)"/g, (...m) => {
-      console.log({ m })
```
Comment (benface) [inline website/pages/en/substreams/[[...slug]].mdx:45]: I committed that? 😅

**Review by benface — APPROVED** [review]
🙏

File: website/pages/en/substreams/[[...slug]].mdx (hunk 1 above)
Comment (B2o5T) [reply website/pages/en/substreams/[[...slug]].mdx:45]: @benface yes 😅

Comment (benface) [reply website/pages/en/substreams/[[...slug]].mdx:45]: I'm ashamed.

**Pull request merged**